### DISCUSSION TITLE
Multiple domain for SMTP

### DISCUSSION THREAD

**Discussion by marirs** [stalwartlabs/smtp-server#46] [Q&A]
Hi,

This is sweet and nice project. Really appreciate for this project.
I'm using only the SMTP server, and how can I have multiple domains with different DKIM keys for each of this.
Where do I need to add the config in the config files? Searched the document couple of days, and could not find it. So thought of asking for help.

Thanks

**Comment by mdecimus** [comment]
Hi, here is the documentation to enable multiple DKIM signatures:

https://stalw.art/docs/smtp/authentication/dkim/sign/#multiple-domains

**Comment by marirs** [comment]
Thanks
